src/mlpack/methods/amf/init_rules/given_init.hpp

```diff
@@ -49,11 +49,25 @@ class GivenInitialization
    * @param H H matrix, to be filled with random noise.
    */
   template<typename MatType>
-  inline void Initialize(const MatType& /* V */,
-                         const size_t /* r */,
+  inline void Initialize(const MatType& V,
+                         const size_t r,
                          arma::mat& W,
                          arma::mat& H)
   {
+    // Make sure the initial W, H matrices have correct size.
+    if (w.n_rows != V.n_rows)
```

Comment:
Do you mind to use ``{}`` for the multiline statement here, same for the following statements, to improve the readability?